### PR TITLE
Add support for missing attributes in PKI UI

### PR DESCRIPTION
**Pull request by cipherboy**
This adds `keyUsage` and `ip_sans` as supported, parsed values for cross signing. Partial support for extKeyUsage was started, but later removed as it was realized the API doesn't support it and doesn't set it by default on Vault-created issuers. While `keyUsage` is present by default, its value isn't controllable via API either, but could be set via an external CSR/issuer, so we still need to validate it.

Also embedded are notes I took on error recovery during cross-signing from a pairing session with Claire.

### CODE REVIEW

File: ui/app/models/pki/certificate/base.js [hunk 1]
```diff
@@ -49,7 +49,8 @@ export default class PkiCertificateBaseModel extends Model {
   // Parsed from cert in serializer
   @attr('number', { formatDate: true }) notValidAfter;
   @attr('number', { formatDate: true }) notValidBefore;
-  @attr('string') uriSans;
+  @attr('string', { label: 'Subject Alternative Names' }) uriSans;
```
Comment (cipherboy) [inline ui/app/models/pki/certificate/base.js:52]: If you were to choose one, I'd maybe do dnsSans as the SANs, and call uri SANs, URI SANs.

Comment (hellobontempo) [reply ui/app/models/pki/certificate/base.js:52]: So label this `URI SANS` and then update `altNames` below (which is dnsNames to `Subject Alternative Names` ?

File: ui/app/models/pki/issuer.js [hunk 2]
```diff
@@ -38,11 +39,6 @@ export default class PkiIssuerModel extends PkiCertificateBaseModel {
   })
   keyId;
 
-  @attr({
```
Comment (hellobontempo) [inline ui/app/models/pki/issuer.js:41]: moved to `PkiCertificateBaseModel`

File: ui/app/models/pki/certificate/base.js (hunk 1 above)
Comment (cipherboy) [reply ui/app/models/pki/certificate/base.js]: Yeah, I think so :-)

File: ui/lib/pki/addon/components/pki-issuer-cross-sign.js [hunk 3]
```diff
@@ -99,6 +99,8 @@ export default class PkiIssuerCrossSign extends Component {
   @action
   async crossSignIntermediate(intMount, intName, newCrossSignedIssuer) {
     // 1. Fetch issuer we want to sign
+    //
+    // What/Recovery: any failure is early enough that you can bail safely/normally.
```
Comment (hellobontempo) [inline ui/lib/pki/addon/components/pki-issuer-cross-sign.js:103]: Claire ToDos for future PR

**Review by hellobontempo — APPROVED** [review]
Thanks Alex!! 🚀

**Pull request merged**